doc/Doxyfile.in

```diff
@@ -497,7 +497,7 @@ GENERATE_HTMLHELP = @GENERATE_HTMLHELP@
 # The HTML Help Workshop contains a compiler that can convert all HTML output generated by doxygen into a single compressed HTML file (.chm). Compressed HTML files are now used as the Windows 98 help format, and will replace the old Windows help format (.hlp) on all Windows platforms in the future. Compressed HTML files also contain an index, a table of contents, and you can search for words in the documentation. The HTML workshop also contains a viewer for compressed HTML files.
 
 
-HHC_LOCATION = ../hhc.exe
+HHC_LOCATION = @HTML_HELP_COMPILER@

```

Comment:
You need quotes around the variable here, in case there are spaces in the path.

Reply:
Thanks, fixed!